packages/core/test/algorithm.test.ts

```diff
@@ -1,95 +0,0 @@
-import { Machine } from '../src';
```

Comment:
as `getConfiguration` is not part of the public API I thought it's not really that relevant for us to test this implementation detail, can revert this deletion if you prefer to keep it